src/odemis/gui/cont/tabs.py

```diff
@@ -345,6 +345,8 @@ def __init__(self, name, button, panel, main_frame, main_data):
         """
 
         tab_data = guimod.LocalizationGUIData(main_data)
+        self._tab_data_model = tab_data
+        self._panel = panel
```

Comment:
That shouldn't be needed. The super() already stores these values in .tab_data_model and .panel.